notebooks/metagenomics_pipeline_iSeqnorm.ipynb

```diff
@@ -1350,6 +1350,13 @@
     "\n",
     "!head {picklist_fp}"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```

Comment:
This looks like an empty cell, just highlighting it in case is not actually needed.

Reply:
Ooo, interesting!  I think this came from the stuff I back-integrated from master, since I've never actually worked on this particular notebook.  

I'd actually like to retire this notebook, because having two different metag notebooks--one of which is being maintained and one of which is *not*--available as templates for the wet lab seems like needlessly tempting fate :) Do we have any kind of procedure for retiring notebooks, or do we just delete them and let git take care of remembering what they used to be?

Reply:
Just remove, we can always bring back if necessary.

Reply:
Great!  Currently waiting for explicit confirmation from the wet lab that they are not using the old one anymore; then I'll zap it :)

Reply:
If this is the only change I will go ahead an approve and we can merge once Antonio approves.